lib/src/client/http2_connection.dart

```diff
@@ -319,7 +315,7 @@ class _SocketTransportConnector implements ClientTransportConnector {
   final Object _host;
```

Comment:
I guess it is worth adding a comment here that `host` is either `InternetAddress` or `String`.

And an assertion into constructor.

Reply:
Done